frontend/index.html

```
@@ -15,56 +15,89 @@
-          <label><input type="checkbox" value="lawson" checked />Lawson</label>
-          <label><input type="checkbox" value="familymart" checked />FamilyMart</label>
-          <label><input type="checkbox" value="daily_yamazaki" checked />Daily Yamazaki</label>
-          <label><input type="checkbox" value="ministop" checked />MINISTOP</label>
-          <label><input type="checkbox" value="michi_no_eki" checked />道の駅</label>
+        <fieldset class="distance-ladder">
+          <legend>近傍距離</legend>
+          <div class="distance-header">
+            <div class="distance-current" id="distance-current">1km</div>
+          </div>
+          <input
+            id="distance-threshold"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)